src/test/java/org/openpnp/libuvc4j/test/TestEnumerateCameras.java

```diff
@@ -39,4 +39,13 @@ public static String getString(Pointer<Byte> p) {
 		}
 		return p.getCString();
 	}
+
+	public static String getHexString(int p) {
```

Comment:
yes, much more elegant to use String.format as suggested... am making the change and will open pull request